src/documenti/esterni/AdR/contenuti/casi_uso.tex

```diff
@@ -94,53 +110,41 @@ \section{UC2 - Selezione tipo di grafico}
      \end{enumerate}
  \end{itemize}
 
- \subsection{UC2.1 - Selezionato Scatter Plot}
+ \subsection{UC4.1 - Selezionato Scatter Plot}
```

Comment:
Inserire quattro sottosezioni identiche in cui cambia solo il nome penso sia inutile, io terrei la generalizzazione nel caso d'uso principale, con magari una descrizione simile a quella fatta nei sotto casi e toglierei la descrizione distinta per ogni sotto caso

Reply:
Sono d'accordo sul fatto che inserire le sottosezioni sia superfluo; però mi è sembrato di capire che sia meglio tenerle in quanto se non ci fossero non si potrebbero inserire i grafici nell'UML diagram. Perciò io direi che potremmo discuterne meglio dopo durante la riunione e in caso chiedere a Cardin quando avremo l'incontro.

Reply:
Ho rimosso le sottosezioni e ho aggiunto i sottocasi (UC4.1,..) nella generalizzazione per evitare che l'immagine non avesse corrispondenze con l'UML